aut-apps.sh

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env bash
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root" 1>&2
+   exit 1
+fi
+function t-d-uninstaller {
+rm -rf $tmp_dir
+echo "Temporary directory uninstalled"
+exit 1
+}
+tmp_dir=$(mktemp -d -t pub_key-XXXXXX)
+trap t-d-uninstaller SIGINT SIGTERM
```

Comment:
Add EXIT here as well
```suggestion
trap t-d-uninstaller SIGINT SIGTERM EXIT
```